kodilla-patterns2/src/main/java/com/kodillapatterns2/observer/homework/Homework.java

```diff
@@ -0,0 +1,22 @@
+package com.kodillapatterns2.observer.homework;
+
+public class Homework {
+    private String taskName;
+    private String description;
+
+    public Homework(String taskName) {
+        this.taskName = taskName;
+    }
+
+    public void setDescription(String description) {
+        this.description = description;
+    }
+
+    public String getTaskName() {
+        return taskName;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+}
```

Comment:
jest Homework jest niemutowalny to powinien miec konstrutor z 2 parametrami i 2 gettery. Jesli mutowalny, to 2 settery i 2 getter. Raczej nie robi sie takich dziwactw - czesciowo mutowalny, czesciowo nie

Reply:
Poprawione, w konstruktorze są 2 parametry i 2 gettery